include/messages.h

```diff
@@ -329,6 +346,12 @@ struct MLSPlaintext
   bytes commit_auth_data() const;
 
   TLS_SERIALIZABLE(group_id, epoch, sender, authenticated_data, content, signature);
+  TLS_TRAITS(tls::vector<1>,
+             tls::pass,
+             tls::pass,
+             tls::vector<4>,
+             tls::variant<ContentType>,
+             tls::vector<2>)
```

Comment:
should this be std::variant for ContentType ( IIUC we are getting rid of tls::variant)